config/locales/views/mailer/hi.yml

```diff
@@ -0,0 +1,39 @@
+en:
+  mailer:
+    welcome_heading: "सर्किटवर्स में आपका स्वागत है, %{name}"
+    keep_building: "निर्माण करते रहे"
+    view_project: "परियोजना देखें"
+  group_mailer:
+    group_created_html: "आपने सफलतापूर्वक एक नया समूह बना लिया है, <span class='mailer-highlighted-text'> %{name} </span>।"
+    view_group: "समूह देखें"
+    member_added_to_group_html: "आपको <span class='mailer-highlighted-text'>%{group_name}</span> नाम के एक नए समूह में जोड़ा गया है। <br>
+    समूह का मार्गदर्शन <span class='mailer-highlighted-text'>%{user_name}</span> द्वारा किया जाता है।"
+  assignment_mailer:
+    assignment_created_html: "<span class='mailer-highlighted-text'>%{name} (%{deadline})</span> नाम का एक की नयी असाइनमेंट <span class='mailer-highlighted-text'> %{group_name} में असाइन की की गयी है </span>. <br>
```

Comment:
Please check the sentence structure again and update this. It has multiple same words like की की and grammatical mistakes.